vorestation.dme

```diff
@@ -1028,6 +1029,8 @@
 #include "code\game\objects\effects\decals\Cleanable\tracks.dm"
 #include "code\game\objects\effects\decals\posters\bs12.dm"
 #include "code\game\objects\effects\decals\posters\polarisposters.dm"
+#include "code\game\objects\effects\decals\posters\polarisposters_vr.dm"
+#include "code\game\objects\effects\decals\posters\tgposters.dm"

```

Comment:
Huh, tgposters wasn't checked?

Reply:
Nope! I decided to check it when I saw that oversight.